Dockerfile

```diff
@@ -1,12 +1,13 @@
 FROM sameersbn/ubuntu:14.04.20170503
 MAINTAINER sameer@damagehead.com
 
-ENV GITLAB_VERSION=9.2.7 \
+ENV GITLAB_VERSION=9.3.0 \
     RUBY_VERSION=2.3 \
-    GOLANG_VERSION=1.6.3 \
-    GITLAB_SHELL_VERSION=5.0.4 \
-    GITLAB_WORKHORSE_VERSION=2.0.0 \
-    GITLAB_PAGES_VERSION=0.4.2 \
+    GOLANG_VERSION=1.8.3 \
```

Comment:
Yeah. I want only fine grained commits for theses changes. Then we get a simple history that everybody can understand directly and doesn't need to check every upgrade commit. I will make a separate commit for every release. So from this point we could see which changes happen between a release.